.github/workflows/canary-release.yml

```diff
@@ -1,6 +1,12 @@
 name: Canary Release
 
 on:
+  workflow_call:
+  secrets:
+    GITHUB_TOKEN:
```

Comment:
Unrelated to your change, but maybe we can align the name here and use `PUBLISH_GITHUB_TOKEN`?

Reply:
Agreed, change pushed.